contracts/SCBridgeWallet.sol

```diff
@@ -160,20 +184,32 @@ contract SCBridgeWallet is IAccount {
   ) external view returns (uint256 validationData) {
     bytes memory ownerSig = userOp.signature[0:65];
     // The owner of the wallet must always approve of any user operation to execute on it's behalf
-    require(
-      validateSignature(userOpHash, ownerSig, owner) ==
-        SIG_VALIDATION_SUCCEEDED,
-      "owner must sign"
-    );
+    if (
+      validateSignature(userOpHash, ownerSig, owner) != SIG_VALIDATION_SUCCEEDED
+    ) {
+      return SIG_VALIDATION_FAILED;
```

Comment:
The ERC 4337 spec specifies that the wallet ` SHOULD return SIG_VALIDATION_FAILED (and not revert) on signature mismatch`